### PR TITLE
Fix transaction from and to fields

### PR DESCRIPTION
**Pull request by skosito**
# Description

- Remove all code related to all mapper, including package.json changes
- Fix relationship between transaction and subject accounts (from and to fields)
- Clean up transactions module and adapt unit tests according to those changes

## Related Issue

https://github.com/eea-oasis/baseline/issues/582

## Motivation and Context

Check issue

## How Has This Been Tested

Postman and unit tests.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Request to be added as a Code Owner/Maintainer

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] I commit to abide by the Responsibilities of Code Owners/Maintainers.

### CODE REVIEW

File: examples/bri-3/src/bri/identity/bpiAccounts/agents/bpiAccountsStorage.agent.ts [hunk 1]
```diff
@@ -38,8 +38,8 @@ export class BpiAccountStorageAgent extends PrismaService {
       bpiAccountModel.ownerBpiSubjectAccounts.map((o) => {
         return new BpiSubjectAccount(
```
Comment (Manik-Jain) [inline examples/bri-3/src/bri/identity/bpiAccounts/agents/bpiAccountsStorage.agent.ts:39]: now that the prisma model and domain model are in sync, we can directly invoke mapper(o, BpiSubjectAccount,BpiSubjectAccount)

File: examples/bri-3/src/bri/identity/bpiAccounts/agents/bpiAccountsStorage.agent.ts [hunk 2]
```diff
@@ -63,8 +63,8 @@ export class BpiAccountStorageAgent extends PrismaService {
         bp.ownerBpiSubjectAccounts.map((o) => {
```
Comment (Manik-Jain) [inline examples/bri-3/src/bri/identity/bpiAccounts/agents/bpiAccountsStorage.agent.ts:63]: now that the prisma model and domain model are in sync, we can directly invoke mapper(o, BpiSubjectAccount,BpiSubjectAccount)

File: examples/bri-3/src/bri/identity/bpiSubjectAccounts/agents/bpiSubjectAccountsStorage.agent.ts [hunk 3]
```diff
@@ -28,12 +28,12 @@ export class BpiSubjectAccountStorageAgent extends PrismaService {
     return new BpiSubjectAccount(
```
Comment (Manik-Jain) [inline examples/bri-3/src/bri/identity/bpiSubjectAccounts/agents/bpiSubjectAccountsStorage.agent.ts:28]: now that the prisma model and domain model are in sync, we can directly invoke mapper(bpiSubjectAccountModel, BpiSubjectAccount,BpiSubjectAccount)

File: examples/bri-3/src/bri/identity/bpiSubjectAccounts/agents/bpiSubjectAccountsStorage.agent.ts [hunk 4]
```diff
@@ -42,13 +42,13 @@ export class BpiSubjectAccountStorageAgent extends PrismaService {
 
   async getAllBpiSubjectAccounts(): Promise<BpiSubjectAccount[]> {
     const bpiSubjectAccountsModels = await this.bpiSubjectAccount.findMany({
-      include: { OwnerBpiSubject: true, CreatorBpiSubject: true },
+      include: { ownerBpiSubject: true, creatorBpiSubject: true },
     });
     return bpiSubjectAccountsModels.map((bp) => {
       return new BpiSubjectAccount(
```
Comment (Manik-Jain) [inline examples/bri-3/src/bri/identity/bpiSubjectAccounts/agents/bpiSubjectAccountsStorage.agent.ts:48]: now that the prisma model and domain model are in sync, we can directly invoke mapper(bp, BpiSubjectAccount,BpiSubjectAccount)

File: examples/bri-3/src/bri/identity/bpiSubjectAccounts/agents/bpiSubjectAccountsStorage.agent.ts [hunk 5]
```diff
@@ -62,18 +62,18 @@ export class BpiSubjectAccountStorageAgent extends PrismaService {
         creatorBpiSubjectId: bpiSubjectAccount.creatorBpiSubject.id,
         ownerBpiSubjectId: bpiSubjectAccount.ownerBpiSubject.id,
       },
-      include: { OwnerBpiSubject: true, CreatorBpiSubject: true },
+      include: { ownerBpiSubject: true, creatorBpiSubject: true },
     });
 
     return new BpiSubjectAccount(
```
Comment (Manik-Jain) [inline examples/bri-3/src/bri/identity/bpiSubjectAccounts/agents/bpiSubjectAccountsStorage.agent.ts:68]: now that the prisma model and domain model are in sync, we can directly invoke mapper(newBpiSubjectAccountModel, BpiSubjectAccount,BpiSubjectAccount)

File: examples/bri-3/src/bri/identity/bpiSubjectAccounts/agents/bpiSubjectAccountsStorage.agent.ts [hunk 6]
```diff
@@ -89,18 +89,18 @@ export class BpiSubjectAccountStorageAgent extends PrismaService {
...
```
Comment (Manik-Jain) [inline examples/bri-3/src/bri/identity/bpiSubjectAccounts/agents/bpiSubjectAccountsStorage.agent.ts:95]: now that the prisma model and domain model are in sync, we can directly invoke mapper(newBpiSubjectAccountModel, BpiSubjectAccount,BpiSubjectAccount)

File: examples/bri-3/src/bri/identity/bpiAccounts/agents/bpiAccountsStorage.agent.ts (hunk 1 above)
Comment (skosito) [reply examples/bri-3/src/bri/identity/bpiAccounts/agents/bpiAccountsStorage.agent.ts:39]: i am pretty sure i got all of these in this PR https://github.com/eea-oasis/baseline/pull/585 but i will double check

**Review by Manik-Jain — APPROVED** [review]

**Review by ognjenkurtic — APPROVED** [review]

**Pull request merged**